### ISSUE TITLE
Show correct time of incoming symptoms via SORMAS-SB

### ISSUE THREAD

**Issue by soja-hzi** [hzi-braunschweig/SORMAS-Project#4674]
### Situation Description
As a health department employee (usually contact supervisor but also all user roles that can see the contact menu) I want to see the correct time at which the contact (and in the future case) person submitted the symptoms via the digital symptom diary SORMAS-SB. Currently, every submission is dated to 1am (see screenshot). The date ist correct. 

![image](https://user-images.githubusercontent.com/77488330/110322535-8dff8800-8013-11eb-8b4e-4a0b56f7d9d0.png)

### Feature Description
Instead of the set time of 1am the correct time at which the contact (and later case) person submitted the daily questionnaire with symptoms via SORMAS-SB is shown in the follow-up visits. It should be the same as if a new visit is added manually.

**Comment by maxiheyner** [comment]
@soja-hzi @ghdoergeloh is PIA actively sending a time to SORMAS?

**Comment by ghdoergeloh** [comment]
Yes, it does:

- `statusDateTime` on `post /sormas-rest/visits-external/person/{uuid}/status` (the current time is submitted)
- **`visitDateTime` on `post /sormas-rest/visits-external/` (submitted is the time, when the questionnaire was issued)**
- other variables in the visit
  - `birthdate`
  - `onsetDate`

I think the bold one is the problem in this issue.

**Comment by kwa20** [comment]
@FredrikSchaeferVitagroup Do you know if this is actually a feature or bug? I can't think of why 1 am is the default value.

**Comment by FredrikSchaeferVitagroup** [comment]
@kwa20 This is unintended behaviour. I doubt that this is a bug in SORMAS, though.
When posting the following visit to SORMAS
```
[
  {
    "personUuid": "XP6LML-AGIEEC-NECG7S-Y42GSM44",
    "disease": "CORONAVIRUS",
    "visitDateTime": "2021-05-12T04:49:05.866Z",
    "visitStatus": "COOPERATIVE",
    "visitRemarks": "string",
    "symptoms": {
      "chillsSweats": "YES"
    }
  }
]
```
the visitDateTime shows as 
![image](https://user-images.githubusercontent.com/67001822/117921749-91541000-b2f1-11eb-9c0a-0e1b4abcab4f.png)
which is correct because the visitDateTime is transmitted as UNIX datetime, which is refers to the UTC timezone. 

@ghdoergeloh I also think the visitDateTime is the problem. Could you please check if it is transmitted according to the example above by PIA?

**Comment by ghdoergeloh** [comment]
Yes I think the bug is that the date_of_issue is used in pia which is the time at which the questionnaire is made available to the user. It should be the date_of_release so the time when the answer was commited. I will create a story in our backlog.

**Comment by soja-hzi** [comment]
@FredrikSchaeferVitagroup when discussing the story in the pia team, we where wondering if sormas would want too save the information at what time the questionnaire is filled in

**Comment by FredrikSchaeferVitagroup** [comment]
@soja-hzi that question has to be answered by HZI members specifying requirements for SORMAS (@kwa20 maybe?).

I personally would expect that PIA sends the questionnaire shortly after a person submitted their result (by clicking "Finally submit" or something alike). I think the most relevant time is transmitted via the `visitDateTime` (which I would expect to be the time the person submitted). 

What would be the value of an additional timestamp in SORMAS showing when a person filled in the questionnaire? Which exact time should that be? The time when a person started filling in? What when the person edited the questionnaire multiple times before finally sumbitting it?

**Comment by kwa20** [comment]
@soja-hzi I would have the same assumptions about the transmission as @FredrikSchaeferVitagroup and therefore agree that the time of entry is not needed because the entries should reach Sormas shortly after they were sent by the person.
